drivers/android/rust_binder.rs

```diff
@@ -35,7 +35,6 @@ module! {
     author: b"Wedson Almeida Filho",
     description: b"Android Binder",
     license: b"GPL v2",
-    params: {},
 }
```

Comment:
This should likely go into a different commit -- or is GitHub confused due to having several commits in the PR? Please rebase to have 1 commit anyway.